README.md

```diff
@@ -1,61 +1,274 @@
 ViSearch PHP SDK
-================
+----
+##Table of Contents
+ 1. [Overview](#1-overview)
+ 2. [Setup](#2-setup)
+ 3. [Initialization](#3-initialization)
+ 4. [Indexing Images](#4-indexing-images)
+	  - 4.1 [Indexing Your First Images](#41-indexing-your-first-images)
+	  - 4.2 [Image with Metadata](#42-image-with-metadata)
+	  - 4.3 [Removing Images](#43-removing-images)
+ 5. [Searching Images](#5-searching-images)
+	  - 5.1 [Pre-indexed Search](#51-pre-indexed-search)
+	  - 5.2 [Color Search](#52-color-search)
+	  - 5.3 [Upload Search](#53-upload-search)
+	    - 5.3.1 [Selection Box](#531-selection-box)
+	    - 5.3.2 [Resizing Settings](#532-resizing-settings)
+ 6. [Search Results](#6-search-results)
+ 7. [Advanced Search Parameters](#7-advanced-search-parameters)
+	  - 7.1 [Retrieving Metadata](#71-retrieving-metadata)
+	  - 7.2 [Filtering Results](#72-filtering-results)
+	  - 7.3 [Result Score](#73-result-score)
+ 8. [Code Samples](#8-code-samples)
+ 
+##1. Overview
+ViSearch is an API that provides accurate, reliable and scalable image search. ViSearch API provides two services ( Data API and Search API) to let the developers prepare image database and perform image searches efficiently. ViSearch API can be easily integrated into your web and mobile applications. More details about ViSearch API can be found in the [documentation](http://www.visenze.com/docs/overview/introduction).
```

Comment:
changes in the second sentence. see https://github.com/visenze/visearch-sdk-java/blob/master/README.md#1-overview